web_src/js/components/ViewFileTreeItem.vue

```diff
@@ -131,6 +142,8 @@ const doGotoSubModule = () => {
 }
 
 .tree-item {
+  color: inherit;
+  text-decoration: inherit;
```

Comment:
You can add one of our link classes from here instead:

https://github.com/go-gitea/gitea/blob/a2ae7c69da638d4cf3801d8bc73567670608b19e/web_src/css/base.css#L253-L256

I guess `silenced` may be the best fit if you never want to highlight.